python/test-requirements.txt

```diff
@@ -1,3 +1,3 @@
 flake8==3.2.1
 mock>=2.0
-testrepository>=0.0.18
+pytest==3.0.7
```

Comment:
This is too late to install tox (since tox installs dependencies from this file). Will add to README.